frame/bags-list/src/benchmarks.rs

```diff
@@ -0,0 +1,97 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Benchmarks for the bags list pallet.
+
+use super::*;
+use crate::list::List;
+use frame_benchmarking::{account, whitelisted_caller};
+use frame_election_provider_support::VoteWeightProvider;
+use frame_support::{assert_ok, traits::Get};
+use frame_system::RawOrigin as SystemOrigin;
+
+frame_benchmarking::benchmarks! {
+	rebag {
+		// An expensive case for rebag-ing:
+		//
+		// - The node to be rebagged should exist as a non-terminal node in a bag with at least
+		//   2 other nodes so both its prev and next are nodes that will need be updated
+		//   when it is removed.
```

Comment:
this is not proper english

Reply:
02f607ae9bab45b7254384ae0e75fe12036b056d